### PR TITLE
factorio, terraria: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**
1.1.109

### CODE REVIEW

File: pages/linux/factorio.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# factorio
+
+> Create and start a headless factorio server.
+> More information: <https://wiki.factorio.com/Multiplayer>.
+
+- Create a new save file:
+
+`{{path/to/}}factorio --create {{path/to/save_file.zip}}`
```
Comment (spageektti) [inline pages/linux/factorio.md:8]:
```suggestion
`{{path/to/factorio}} --create {{path/to/save_file.zip}}`
```
It looks weird when only part is highlighted.

File: pages/linux/factorio.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+- Start a factorio server:
+
+`{{path/to/}}factorio --start-server {{path/to/save_file.zip}}`
```
Comment (spageektti) [inline pages/linux/factorio.md:12]:
```suggestion
`{{path/to/factorio}} --start-server {{path/to/save_file.zip}}`
```
^

**Review by spageektti — APPROVED** [review]

**Comment by Managor** [comment]
An experiment in creating pages for headless game servers. Factorio is usually not installed to your path, which is why I wrote the binary as `path/to/factorio`

File: pages/linux/factorio.md (hunk 1 above)
Comment (Managor) [reply pages/linux/factorio.md:8]: image
Looks alright to me. Having it be partly highlighted would be correct as the binary name stays the same.

File: pages/linux/factorio.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# factorio
+
+> Create and start a headless factorio server.
```
Comment (kbdharun) [inline pages/linux/factorio.md:3]:
```suggestion
> Create and start a headless Factorio server.
```

File: pages/linux/factorio.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+> More information: <https://wiki.factorio.com/Multiplayer>.
+
+- Create a new save file:
+
+`{{path/to/}}factorio --create {{path/to/save_file.zip}}`
+
+- Start a factorio server:
```
Comment (kbdharun) [inline pages/linux/factorio.md:10]:
```suggestion
- Start a new Factorio server:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, LGTM after these suggestions.

File: pages/linux/factorio.md (hunk 4 above)
Comment (Managor) [reply pages/linux/factorio.md:10]: I disagree with the word "new" as you can restart the server you closed yesterday.

**Comment by Managor** [comment]
Since this is an experimental idea, I'll add Terraria server instructions here too.

**Comment by Managor** [comment]
Asking for guidance. For the Terraria server I can use either the TerrariaServer helper script or I can run the TerrariaServer.bin.x86_64 binary directly. I want to know what name should be given to the markdown file.

**Comment by Managor** [comment]
Ah right, OpenTTD is possible to be installed through your package manager.

File: pages/linux/terraria.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+> Create and start a headless Terraria server.
+> More information: <https://terraria.wiki.gg/wiki/Server>.
+
+- Start an interactive server setup:
+
+`{{path/to/TerrariaServer}}`
+
+- Start a Terraria server:
+
+`{{path/to/TerrariaServer}} -world {{path/to/world.wld}}`
```
[the remark this answers is not in the document]
Comment (Managor) [reply pages/linux/terraria.md:12]: I'm against this as it's not the executable that the game comes with.

[the remark this answers is not in the document]
Comment (Managor) [reply pages/linux/terraria.md:12]: The game comes with the helper script so I think TerrariaServer should be enough 
image

**Pull request merged**